server/dbs/__init__.py

```diff
@@ -17,9 +17,10 @@
 #  limitations under the License.
 ##############################################################################
 
-from base import (getDBConnectorClass, getDBConnector,   # noqa
-                  clearDBConnectorCache, FilterOperators,
-                  DatabaseConnectorException)
-import sqlalchemydb  # noqa
-import postgres_sqlalchemy  # noqa
-import mongo  # noqa
+from .base import (   # noqa
+    getDBConnectorClass, getDBConnector, getDBConnectorClassFromDialect,
+    clearDBConnectorCache, FilterOperators, DatabaseConnectorException,
+    databaseFromUri)
+from . import sqlalchemydb  # noqa
+from . import postgres_sqlalchemy  # noqa
+from . import mongo  # noqa
```

Comment:
I'm guessing flake8 complains about these as unused imports, but this is totally valid behavior that is very common. Is there something easy we can do besides silencing flake8 to make it clear these are being imported just to be in the module namespace?